### ISSUE TITLE
How to pass in text instruction?

### ISSUE THREAD

**Issue by jonathonsather-bc** [SudeepDasari/dit-policy#2]
The paper mentions that a text instruction can be encoded into the observation vector, although it is not clear to me how (if?) this is done in the repo. An example of how to do this would be would be appreciated!

**Comment by bbbbbbbbly** [comment]
yes, where the bert gone?

**Comment by Lorenzo-Mazza** [comment]
Yeah also my question is about this. In the paper they claimed they used FiLM modulation for the text instruction...
